src/command/dev/next/mod.rs

```diff
@@ -62,12 +62,6 @@ impl Dev {
 
         let router_config_path = self.opts.supergraph_opts.router_config_path.clone();
 
-        let _config = RunRouterConfig::default()
-            .with_address(router_address)
-            .with_config(&read_file_impl, router_config_path.as_ref())
-            .await
-            .map_err(|err| RoverError::new(anyhow!("{}", err)))?;
-
```

Comment:
This value `_config` was thrown away; computing this had the side effect of throwing here when the filepath is bad. I want to wait until a later point to handle the filepath issue.